tests/test_section.py

```diff
@@ -111,6 +125,22 @@ def it_can_change_its_page_margins(self, margins_set_fixture):
 
     # fixtures -------------------------------------------------------
 
+    @pytest.fixture
+    def Header_(self, request, header_):
+        return class_mock(
+            request, 'docx.section.Header',
+            return_value=header_
+        )
+
+    @pytest.fixture
+    def header_(self, request):
+        return instance_mock(request, Header)
+
+    @pytest.fixture
+    def header_fixture(self, Header_, header_):
+        section = Section(element('w:sectPr'))
+        return section, Header_, header_
```

Comment:
This will need to be just slightly different:

```python
@pytest.fixture
def header_fixture(self, Header_, header_):
    sectPr = element('w:sectPr')
    section = Section(sectPr)
    return section, Header_, sectPr, header_
````

The return values are of course in the same order as they are "unpacked" in the unit test. I order them in the unit test (which I write first generally) in the order they are used. Then I copy that down to the return statement, and create them in the order they appear in the return statement, satisfying prerequisites as necessary. You seem to have picked that up, so just in case you were wondering why things showed up that way :)  I generally find it useful to be as consistent and mechanical as possible; it speeds the development process (like conforming to a kata) and of course makes it easier to follow months later when you look at it again :)